VP_VKD3D_requirements.json

```diff
@@ -0,0 +1,759 @@
+                    "shaderDenormPreserveFloat32": true
+                },
+                "VkPhysicalDeviceSubgroupProperties": {
+                    "subgroupSize": 4,
+                    "supportedStages": ["VK_SHADER_STAGE_COMPUTE_BIT", "VK_SHADER_STAGE_FRAGMENT_BIT"],
+                    "supportedOperations": ["VK_SUBGROUP_FEATURE_ARITHMETIC_BIT", "VK_SUBGROUP_FEATURE_BASIC_BIT", "VK_SUBGROUP_FEATURE_BALLOT_BIT", "VK_SUBGROUP_FEATURE_SHUFFLE_BIT", "VK_SUBGROUP_FEATURE_QUAD_BIT", "VK_SUBGROUP_FEATURE_VOTE_BIT"]
+
+                }
+            }
+        },
+        "d3d12_level_12_2_sm6_2_32_bit_only": {
+            "properties": {
+                "VkPhysicalDeviceVulkan12Properties": {
+                    "denormBehaviorIndependence": "VK_SHADER_FLOAT_CONTROLS_INDEPENDENCE_32_BIT_ONLY",
+                    "shaderDenormFlushToZeroFloat32": true,
+                    "shaderDenormPreserveFloat32": true
+                }
+            }
+        },
+        "d3d12_level_12_2_sm6_2_32_all": {
+            "properties": {
+                "VkPhysicalDeviceVulkan12Properties": {
+                    "denormBehaviorIndependence": "VK_SHADER_FLOAT_CONTROLS_INDEPENDENCE_ALL",
+                    "shaderDenormFlushToZeroFloat32": true,
+                    "shaderDenormPreserveFloat32": true
+                }
+            }
+        },
+        "d3d12_level_12_2_sm6_2_32_bit_only_nv": {
+            "properties": {
+                "VkPhysicalDeviceProperties": {
+                    "vendorID": 4318
+                },
+                "VkPhysicalDeviceVulkan12Properties": {
+                    "denormBehaviorIndependence": "VK_SHADER_FLOAT_CONTROLS_INDEPENDENCE_32_BIT_ONLY"
+                }
+            }
+        },
+        "d3d12_level_12_2_sm6_2_32_all_nv": {
+            "properties": {
+                "VkPhysicalDeviceProperties": {
+                    "vendorID": 4318
+                },
+                "VkPhysicalDeviceVulkan12Properties": {
+                    "denormBehaviorIndependence": "VK_SHADER_FLOAT_CONTROLS_INDEPENDENCE_ALL"
+                }
+            }
+        },
+        "d3d12_level_12_2_sm6_6": {
+            "extensions": {
+                "VK_NV_compute_shader_derivatives": 1
+            },
+            "features": {
+                "VkPhysicalDeviceVulkan12Features": {
+                    "shaderInt8": true,
+                    "shaderBufferInt64Atomics": true
+                },
+                "VkPhysicalDeviceComputeShaderDerivativesFeaturesNV": {
+                    "computeDerivativeGroupLinear": true
+                }
+            },
+            "properties": {
+                "VkPhysicalDeviceVulkan13Properties": {
+                    "requiredSubgroupSizeStages": ["VK_SHADER_STAGE_COMPUTE_BIT"]
+                }
+            }
+        }
+    },
+    "profiles": {
+        "VP_VKD3D_d3d12_level_11_0_baseline": {
+            "version": 1,
+            "api-version": "1.3.204",
```

Comment:
Because `api-version` refers to a specific Vulkan Header version, it indicates what's the minimum Vulkan API revision the Vulkan drivers must supports for the necessary to be potentially exposed and supported. 

204 is the first Vulkan 1.3 Vulkan Header version and this profile only requires Vulkan 1.3 and not additional extensions or capabilities added in following Vulkan Header revision.

This said, you can decide that you want to require new Vulkan API version if you want just to explicitly claim that you don't want to support Vulkan drivers that old.

I used the Profiles JSON schemas here https://schema.khronos.org/vulkan/ to figure out these `api-version` for each profile.